src/Components/CurrencyDetail.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CurrencyDetail = (props) => {
+  const { detail } = props;
+  const details = Object.entries(detail).map(([key, value]) => (
+    <li key={key}>
+      <div className="currency-detail-wrapper">
+        <div>
+          <p className="crypto-name">
+            {key}
+            {' '}
+          </p>
+        </div>
+        <div>
+          {value}
+          {' '}
+        </div>
```

Comment:
- [ ] OPTIONAL - Consider adding margin/padding to the details listed.

<img src="https://user-images.githubusercontent.com/33323077/132945930-1f43ab5a-4278-472d-96fd-8d37004b2a79.png" height="250px"/>